packages/twenty-server/src/app.module.ts

```diff
@@ -25,6 +29,15 @@ import { WorkspaceModule } from './workspace/workspace.module';
     IntegrationsModule,
     CoreModule,
     WorkspaceModule,
+    ThrottlerModule.forRootAsync({
+      useClass: ThrottlerConfigService,
+    }),
+  ],
+  providers: [
+    {
+      provide: APP_GUARD,
```

Comment:
I'm not sure if it's going to work for the flexible schema, @charlesBochet I think we need to test this